### ISSUE TITLE
!skip causes error with no songs remaining in playlist

### ISSUE THREAD

**Issue by Wrunt** [Richardson-Media-House/kirbi-discord-musicplayer#13]
Stack trace:
```Command stop failed :disappointed_relieved:
TypeError: Cannot read property 'end' of undefined
    at Object.process (/home/ec2-user/Kirbi/node_modules/kirbi-discord-musicplayer/musicplayer.js:335:38)
    at checkMessageForCommand (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:141:11)
    at Client.Kirbi.Discord.on.msg (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:158:37)
    at emitOne (events.js:115:13)
    at Client.emit (events.js:210:7)
    at MessageCreateHandler.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
    at WebSocketPacketManager.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:103:65)
    at WebSocketConnection.onPacket (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:330:35)
    at WebSocketConnection.onMessage (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:293:17)
    at WebSocketClient.internalOnMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:103:17)
    at native.client.group.onMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:57:15)
Command skip failed :disappointed_relieved:
TypeError: Cannot read property 'end' of undefined
    at Object.process (/home/ec2-user/Kirbi/node_modules/kirbi-discord-musicplayer/musicplayer.js:212:38)
    at checkMessageForCommand (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:141:11)
    at Client.Kirbi.Discord.on.msg (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:158:37)
    at emitOne (events.js:115:13)
    at Client.emit (events.js:210:7)
    at MessageCreateHandler.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
    at WebSocketPacketManager.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:103:65)
    at WebSocketConnection.onPacket (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:330:35)
    at WebSocketConnection.onMessage (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:293:17)
    at WebSocketClient.internalOnMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:103:17)
    at native.client.group.onMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:57:15)```

**Comment by Wrunt** [comment]
Stack trace:
```Command stop failed :disappointed_relieved:
TypeError: Cannot read property 'end' of undefined
    at Object.process (/home/ec2-user/Kirbi/node_modules/kirbi-discord-musicplayer/musicplayer.js:335:38)
    at checkMessageForCommand (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:141:11)
    at Client.Kirbi.Discord.on.msg (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:158:37)
    at emitOne (events.js:115:13)
    at Client.emit (events.js:210:7)
    at MessageCreateHandler.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
    at WebSocketPacketManager.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:103:65)
    at WebSocketConnection.onPacket (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:330:35)
    at WebSocketConnection.onMessage (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:293:17)
    at WebSocketClient.internalOnMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:103:17)
    at native.client.group.onMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:57:15)
Command skip failed :disappointed_relieved:
TypeError: Cannot read property 'end' of undefined
    at Object.process (/home/ec2-user/Kirbi/node_modules/kirbi-discord-musicplayer/musicplayer.js:212:38)
    at checkMessageForCommand (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:141:11)
    at Client.Kirbi.Discord.on.msg (/home/ec2-user/Kirbi/node_modules/kirbi-discord/lib/onEvent/message.js:158:37)
    at emitOne (events.js:115:13)
    at Client.emit (events.js:210:7)
    at MessageCreateHandler.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
    at WebSocketPacketManager.handle (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:103:65)
    at WebSocketConnection.onPacket (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:330:35)
    at WebSocketConnection.onMessage (/home/ec2-user/Kirbi/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:293:17)
    at WebSocketClient.internalOnMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:103:17)
    at native.client.group.onMessage (/home/ec2-user/Kirbi/node_modules/uws/uws.js:57:15)```

**Comment by naterchrdsn** [comment]
resolved with #15
https://github.com/Richardson-Media-House/kirbi-discord-musicplayer/pull/15
